### ISSUE TITLE
Trouble installing 2.3

### ISSUE THREAD

**Issue by imkevinxu** [carljm/django-model-utils#178]
Getting the following error when trying to upgrade to 2.3, any ideas?

```
Running setup.py install for django-model-utils
Complete output from command /app/.heroku/python/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-J22Nrq/django-model-utils/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-473IRE-record/install-record.txt --single-version-externally-managed --compile:
running install
running build
running compile_translations
CommandError: Can't find msgfmt. Make sure you have GNU gettext tools 0.15 or newer installed.

----------------------------------------
Rolling back uninstall of django-model-utils
Command "/app/.heroku/python/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-build-J22Nrq/django-model-utils/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-473IRE-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-J22Nrq/django-model-utils
```

**Comment by carljm** [comment]
Hrm, the first translation was recently added, and some things were added to `setup.py` that might be causing problems. I'll look into it, thanks for the report.

**Comment by joeweiss** [comment]
Getting the same error here. Thanks.

**Comment by approxit** [comment]
Another problem here.

Requirements installation (pip install -r) is happening before even project files are present, but `ENV`'s are set. Triggering `compilemessages` by `model_utils` can't load project `DJANGO_SETTINGS_MODULE`. Even after reordering a little bit, installing `model_utils` with project files, it fails to load project settings.

Compiling messages on package install is a bad idea. I have not seen such a practice in any other package. Now we know why.

**Comment by richardasaurus** [comment]
On ubuntu, installing `gettext` fixes this

**Comment by carljm** [comment]
I just pushed a commit to fix this, and will issue a 2.3.1 release. Sorry everyone for the hassle of the broken release; my fault for not reviewing that PR closely enough to realize the implications of the `setup.py` changes.

**Comment by carljm** [comment]
2.3.1 is released. Sorry again for the breakage.
